easybuild/easyconfigs/m/magma/make.inc

```diff
@@ -0,0 +1,49 @@
+#//////////////////////////////////////////////////////////////////////////////
```

Comment:
ah, sorry, wrong context... >_<

Look for `Environment variable .* set to` in the log, for example:

```
 == 2015-04-01 21:56:22,550 main.environment INFO Environment variable CXX set to g++
```